lib/tagging/linkables.rb

```diff
@@ -0,0 +1,43 @@
+# Linkables
```

Comment:
Perhaps it's best to copy the new version of `Linkables` from content-tagger so we don't have to use the class variables:

https://github.com/alphagov/content-tagger/blob/master/lib/linkables.rb